roles/manage_touchstone_tools/README.md

```diff
@@ -0,0 +1,307 @@
+
+Below is an example of how to put `start_system_stats` in your playbook.
+
+```yaml
+- name: Include start_system_stats tasks to start sysstat data collection
+  ansible.builtin.include_role:
+    name: manage_touchstone_tools
+    tasks_from: start_system_stats
+  vars:
+    ts_output_dir: "/var/log/touchstone/system_stats"
+    sec_bw_sample: 30
+```
+
+### `stop_system_stats`
+
+This task stops the collection of system statistics.
+
+#### `ts_output_dir`
+
+The output directory for the touchstone-tools data collection. This must be a 
+directory that does not already exist.
+ 
+#### How to include `stop_system_stats` in your playbook
+
+Below is an example of how to put `stop_system_stats` in your playbook.
+
+```yaml
+- name: Include stop_system_stats tasks to stop sysstat data collection
+  ansible.builtin.include_role:
+    name: manage_touchstone_tools
+    tasks_from: stop_system_stats
+  vars:
+    ts_output_dir: "/var/log/touchstone/system_stats"
+```
+
+### `start_db_stats`
+
+This task starts the collection of database statistics.
+
+#### `ts_output_dir`
+
+The output directory for the touchstone-tools database data collection.
+
+#### `sec_bw_sample`
+
+The number of seconds between samples, default 60.
+
+#### `db_name`
+
+The database name to connect to.
+
+#### `db_server_name`
+
+Database server host or docker directory. Same as `-h` in `psql` commands.
+
+#### `db_port`
+
+Database server port. Same as `-p` in `psql` commands.
+
+#### `db_user`
+
+The database connection username. Same as `-U` in `psql` commands.
+ 
+#### How to include `start_db_stats` in your playbook
+
+Below is an example of how to put `start_db_stats` in your playbook.
+
+```yaml
+- name: Include start_db_stats tasks to start pgsql stats data collection
+  ansible.builtin.include_role:
+    name: manage_touchstone_tools
+    tasks_from: start_db_stats
+  vars:
+    ts_output_dir: "/var/log/touchstone/db_stats"
+    sec_bw_sample: 15
+    db_name: "postgres"
+    db_server_hostname: "primary1"
+    db_server_port: 5432
+    db_user: "postgres"
+```
+
+### `stop_db_stats`
+
+This task stops the collection of database statistics.
+
+#### `ts_output_dir`
+
+The output directory for the touchstone-tools database data collection.
+
+#### `db_name`
+
+The database name to connect to.
+
+#### `db_server_name`
+
+Database server host or docker directory. Same as `-h` in `psql` commands.
+
+#### `db_port`
+
+Database server port. Same as `-p` in `psql` commands.
+
+#### `db_user`
+
+The database connection username. Same as `-U` in `psql` commands.
+ 
+#### How to include `stop_db_stats` in your playbook
+
+Below is an example of how to put `stop_db_stats` in your playbook.
+
+```yaml
+- name: Include stop_db_stats tasks to stop pgsql stats data collection
+  ansible.builtin.include_role:
+    name: manage_touchstone_tools
+    tasks_from: stop_db_stats
+  vars:
+    ts_output_dir: "/var/log/touchstone/db_stats"
```

Comment:
The `ts_output_dir` should be the only var required for stopping.  The others should not be needed.  I wanted to double check if you found otherwise.  If so, it's a bug. :)